### PR TITLE
Fix HelperTest on edge-rails

### PR DESCRIPTION
**Pull request by yhirano55**
Fixed helper_test.rb on edge-rails. Related #944 

image

### CODE REVIEW

File: test/helper_test.rb [hunk 1]
```diff
@@ -1,39 +1,44 @@
...
-    @view.extend Webpacker::Helper
+    @request = Class.new do
+      def send_early_hints(links) end
+    end.new
   end
 
   def test_asset_pack_path
-    assert_equal @view.asset_pack_path("bootstrap.js"), "/packs/bootstrap-300631c4f0e0f9c865bc.js"
-    assert_equal @view.asset_pack_path("bootstrap.css"), "/packs/bootstrap-c38deda30895059837cf.css"
+    assert_equal "/packs/bootstrap-300631c4f0e0f9c865bc.js", asset_pack_path("bootstrap.js")
```
Comment (gauravtiwari) [inline test/helper_test.rb]: Should be `expected` and `actual` right? 

```rb
assert_equal asset_pack_path("bootstrap.js"), "/packs/bootstrap-300631c4f0e0f9c865bc.js"
```

File: test/helper_test.rb [hunk 2]
```diff
@@ -1,39 +1,44 @@
...
+    assert_equal "/packs/bootstrap-c38deda30895059837cf.css", asset_pack_path("bootstrap.css")
   end
 
   def test_javascript_pack_tag
     assert_equal \
       %(<script src="/packs/bootstrap-300631c4f0e0f9c865bc.js"></script>),
-      @view.javascript_pack_tag("bootstrap.js")
+      javascript_pack_tag("bootstrap.js")
```
Comment (gauravtiwari) [inline test/helper_test.rb]: Ahh, looks like it's written the opposite way everywhere

Comment (yhirano55) [reply test/helper_test.rb]: Oh, My understanding is... "expected" is left position, and "actual" is right position.

https://github.com/seattlerb/minitest/blob/master/lib/minitest/assertions.rb#L159-L188

But If I need to modify all test cases, I'll do it. I'll respect to your opinion :)

Comment (gauravtiwari) [reply test/helper_test.rb]: Ahh, yes that's right 👍  kinda of confusing though, depending on how one interprets `expected` and `actual`

**Comment by gauravtiwari** [comment]
Thanks 👍  🍰

**Pull request merged**